### DISCUSSION TITLE
My commits dont count as contribution even though it updates the repository

### DISCUSSION THREAD

**Discussion by 1804049Shorna** [community/community#71533] [Discussions] [answered]
### Select Topic Area

Question

### Body

I did commited and push my code changes to github repository from VS code ,but still it doesnt count as my contribution ,but in repository commit I can see the commit ,why ?

here I have doubled check the solution :
1.My email connected repository is correct 
2. My repository is public not private 
3. I have pushed on master branch 
4. My repository updates the changes 
5. The changes are major nor minor 

I have waited couple of days to count as contribution 
still I cant see any green box

**Comment by TankEngine-ish** [comment]
Is master your default branch at the moment? If not, the changes may not appear on your graph until
you set it to default or perform a merge.

**Reply by 1804049Shorna** [reply]
https://github.com/1804049Shorna/todo

This is my repo link ,and master branch is default ,still not working

**Comment by Vijaykv5** [comment, marked as answer]
I can see the green box graph is not showing your contribution @1804049Shorna as well as in repo, your commits are reflected too ..
Just go through this [Link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#common-reasons-that-contributions-are-not-counted) and even if you don't find solution, chat with github virtual agent in above link

**Comment by tobias-barakaa** [comment]
Me too, GitHub doesn't recognize my contribution unless I change something directly from github it doesn't end there after I change something directly from my github even if I push as more as twenty times the green bar doesn't turn dark it started a month ago.

**Reply by kinjalsri** [reply]
same issue, i am facing these days!
